test/fixtures/doctype-with-embedded-expr/output.astro

```diff
@@ -16,8 +16,10 @@ const colors = ["red", "yellow", "blue"];
 
     {"I'm some super long text and oh boy I sure do hope this formatter doesn't break me!"}
 
-    {colors.map((color) => <div>
+    {colors.map((color) => (
```

Comment:
What caused this formatting change?

Reply:
[This change](https://github.com/snowpackjs/prettier-plugin-astro/pull/47/files#diff-35264771cf86f41ef7b7d56b764142b299a8bd2a972b2316e4e3e92e13056500R397):
```js
case 'MustacheTag':
  return [
    '{',
    printJS(path, print, 'expression', {
      forceSingleLine: isInsideQuotedAttribute(path), // <- this line
      forceSingleQuote: opts.jsxSingleQuote,
    }),
    '}',
  ];
```

Reply:
Why would that cause the wrap in parentheses? Isn't it already not one line?

Reply:
I feel like the parentheses are unnecessary since it is a single element, unlike how in React you might need something like this, I think the original text compiles fine in Astro...

(This is fine too, I'm just being particular about trying to get it right in a minimal way?)

Reply:
This is how jsx and svelte are formating the same code. I think that formating the same way [prettier](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEACSAbCAnAzgOgFsBDABwApzMcBKVAXgD5UAeAEwEsA3RgHSlSChglgGEIWbKijFCcesGrYAvqgD0fAcNZrOPGiAA0ICKRgdouZKGLZsEAO4AFWwispiGB8QCeV4wBG2MRgANZwMADKsnAAMhxQcMgAZp64cIHBYRGRpCEJAObIMNgArhkgcIQBcGxstbHEUAWlxAVwAGI4JDDmzcggxKUwEEYgABYwhBgA6uMc8Lh5YHCRbgvcCz4DYLj+IAnp2DBOwQUkKWkVAFa4AB6RhRhwAIqlEPCXGOnGeXhw2AGAWINQwY1I2ASMBmHDYMHGyAAHAAGX72dIzYKkAYQuBHLhJYwAR3e8FOpncg1wAFpErVamNsHASRwmac2hckKlvhV0oQOMUyrynq9SUkuVdjDAQTC4QikAAmKXBDgYQriQicyq4ACsY1K6QAKiD3NyfiAuOUAJJQeqwSJgSFmACCtsiMB8zy+6WUyiAA) does will be expected output

Reply:
https://github.com/prettier/prettier/issues/11791 :p added an issue requesting Svelte support in the prettier playground.

 It'd be nice to see that easier since there are a few formatting differences since Svelte and Astro are closer in some ways to formatting XML than JSX - JSX is Javascript + HTML whereas Astro aims to be HTML + Javascript I feel. For example, `class` is a valid attribute name and you don't need a single top-level element to scope the component you have.

Reply:
https://github.com/jasikpark/unofficial-distractible-scoreboard/commit/950677703ad75f07fc60d9708399ef0f5f410ed7 seems to have added () in a lot of places where it seems it was unnecessary... I'm not sure why exactly.

Reply:
It behaves like when an arrow function has only one parameter and prettier adds the () anyways even though it's not technically needed. I guess we can make an option to disable it, but I think the Astro plugin should behave as close as prettier as possible. See [this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEwDOEBO8AmBlSTONAOgFsBDABwAoACGtQuAGjqoBsKw4yEYAlHQC8APjoAeAIIcOAFQCWfAljh0mq4emYBfdlx59YWzt1789AenECdAHSh0nzl6+cPIUNDDoAzCBAidI5iwSAsIBBUMArQaMigFJiYEADuAApJCPEoFBypFACe8REARpjcANZwMHgUfAAyClBwyL55aKwg5VU1eFTczQDmyDCYAK5dvKVwODizDRRQQ+MUQ3AAYliUMDHLyCAU4zAQ4SAAFjBkHADq5wrwaAM8eNkPCgBuD4UHYGglIGanWw6QqQ0obQ6XQAVmgAB54YYcOAARXGEHgkI4nQiA0wwIOpQoMw4ZyomGaMBuChwMHOyAAHAAGXEpTo3CpUA7k4hwTAfVoRACO6PgoKiOUOaAAtC1ZrMzkQRQoiKC1hCkO1sV1OmQFKMJjqkajRa1NVCIjBidTafSkAAmS0VBQcYYAYQgZA1IGIAFYzuNOnJiTktTiQB9JgBJKDzWAECnRKSxvAwQrIrGdHQ6IA) example.

Reply:
I'll look through the prettier PR's & whatnot I guess